src/renderer/app-bridge.ts

```diff
@@ -80,10 +78,11 @@ export class AppBridge {
   constructor() {
     // starts with corporate pod and
     // will be updated with the global config url
-    const currentWindow = remote.getCurrentWindow();
+    // TODO: use ipc sync to get the origin
+    // const currentWindow = remote.getCurrentWindow();
     // @ts-ignore
-    this.origin = currentWindow.origin || '';
-    // this.origin = '*'; // DEMO-APP: Comment this line back in only to test demo-app - DO NOT COMMIT
+    // this.origin = currentWindow.origin || '';
+    this.origin = '*'; // DEMO-APP: Comment this line back in only to test demo-app - DO NOT COMMIT
```

Comment:
I thought we shouldn't commit this line 😅

Reply:
I have temporarily removed the comment here and added a todo. I will investigate to use IPC sync here to get the origin instead of remote.